Firestore/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 - [changed] Firestore no longer has a direct dependency on FirebaseAuth.
+- [changed] property includeMetadataChanges is now hidden in FIRDocumentListenOptions
```

Comment:
From a user's point of view it doesn't really matter that this is now an internal property. From their point of view we just removed it.

This would be better phrased as:

Removed the includeMetadataChanges property in FIRDocumentListenOptions to avoid confusion with the factory method of the same name.

Reply:
done